panedr/panedr.py

```diff
@@ -75,7 +75,7 @@
 Enxnm = collections.namedtuple('Enxnm', 'name unit')
 ENX_VERSION = 5
 
-__all__ = ['edr_to_df']
+__all__ = ['edr_to_df', 'edr_to_dict']
```

Comment:
I don't see a reason not to add `read_edr` here as well.

Reply:
I saw `read_edr` as merely providing data for the two user-exposed functions `edr_to_df` and `edr_to_dict`, so that the user should never need to call `read_edr` itself directly. I am not sure if the return values of this function are of use to a user, but yeah, that's not really a reason to add it here.